conversions/Backwards/src/main/java/com/agentdid127/resourcepack/backwards/BackwardsPackConverter.java

```diff
@@ -15,71 +15,91 @@
+    public BackwardsPackConverter(String from, String to, String light, boolean minify, Path input, boolean debug,
+            PrintStream out) {
         GsonBuilder gsonBuilder = new GsonBuilder().disableHtmlEscaping();
-        if (!minify) gsonBuilder.setPrettyPrinting();
+        if (!minify)
+            gsonBuilder.setPrettyPrinting();
         gson = gsonBuilder.create();
         DEBUG = debug;
         PackConverter.out = out;
         this.INPUT_DIR = input;
-
         log(from);
         log(to);
-
         converterRunner(from, to, light);
-        
     }
-    
+
     protected void converterRunner(String from, String to, String light) {
-        // this needs to be run first, other converters might reference new directory names
-        this.registerConverter(new NameConverter(this, Util.getVersionProtocol(gson, from), Util.getVersionProtocol(gson, to)));
+        // this needs to be run first, other converters might reference new directory
+        // names
+        this.registerConverter(
+                new NameConverter(this, Util.getVersionProtocol(gson, from), Util.getVersionProtocol(gson, to)));
+        
         this.registerConverter(new PackMetaConverter(this, Util.getVersionProtocol(gson, to)));
-        this.registerConverter(new DeleteFileConverter(this, Util.getVersionProtocol(gson, from), Util.getVersionProtocol(gson, to)));
+        
+        this.registerConverter(
+                new DeleteFileConverter(this, Util.getVersionProtocol(gson, from), Util.getVersionProtocol(gson, to)));
+
+        // TODO: backwards title converter for going from 1.20 to anything below
 
-        if (Util.getVersionProtocol(gson, from) >= Util.getVersionProtocol(gson, "1.19.4") && Util.getVersionProtocol(gson, to) < Util.getVersionProtocol(gson, "1.19.4")) {
+        if (Util.getVersionProtocol(gson, from) >= Util.getVersionProtocol(gson, "1.19.4")
+                && Util.getVersionProtocol(gson, to) < Util.getVersionProtocol(gson, "1.19.4")) {
             this.registerConverter(new EnchantPathConverter(this));
         }
-        if (Util.getVersionProtocol(gson, from) > Util.getVersionProtocol(gson, "1.18") && Util.getVersionProtocol(gson, to) <= Util.getVersionProtocol(gson, "1.18")) {
+
+        if (Util.getVersionProtocol(gson, from) > Util.getVersionProtocol(gson, "1.18")
+                && Util.getVersionProtocol(gson, to) <= Util.getVersionProtocol(gson, "1.18")) {
             this.registerConverter(new ParticleConverter(this));
             this.registerConverter(new InventoryConverter(this));
         }
-        if (Util.getVersionProtocol(gson, from) >= Util.getVersionProtocol(gson, "1.13") && Util.getVersionProtocol(gson, to) <= Util.getVersionProtocol(gson, "1.14.4")) {
+
+        if (Util.getVersionProtocol(gson, from) >= Util.getVersionProtocol(gson, "1.13")
+                && Util.getVersionProtocol(gson, to) <= Util.getVersionProtocol(gson, "1.14.4")) {
```

Comment:
&& doesn't need to be on a second line either.

Reply:
formatter